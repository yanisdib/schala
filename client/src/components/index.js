// COMPONENTS EXPORTS

export { default as Header } from './Header/Header';

export { default as Button } from './Button/Button';
export { default as MenuButton } from './MenuButton/MenuButton';
export { default as DisplayIdentifiedButton } from './DisplayIdentifiedButton/DisplayIdentifiedButton';

export { default as Tag } from './Tag/Tag';
export { default as Pointer } from './Pointer/Pointer';
export { default as Bubble } from './Bubble/Bubble';

export { default as PictureIdentifier } from './PictureIdentifier/PictureIdentifier';