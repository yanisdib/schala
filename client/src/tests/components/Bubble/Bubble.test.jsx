import { shallow } from 'enzyme';

import { Bubble } from '../../../components';

describe('Bubble component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<Bubble />))

    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot();
    })
})