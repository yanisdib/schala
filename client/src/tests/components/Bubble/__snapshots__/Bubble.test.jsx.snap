// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bubble component renders correctly 1`] = `
<div
  className="sc-pGacB idJyiZ"
/>
`;
